gradio/utils.py

```diff
@@ -680,3 +680,28 @@ def validate_url(possible_url: str) -> bool:
 
 def is_update(val):
     return type(val) is dict and "update" in val.get("__type__", "")
+
+
+def mount_gradio_app(
+    app: fastapi.FastAPI, gradio_app: App, server_name: str, port: str, path: str
```

Comment:
Can we make `port` an `int` to make it consistent the type of the server port in other parts of the codebase?